meta-intel-edison-bsp/recipes-bsp/u-boot/u-boot-common_2018.05.inc

```diff
@@ -25,17 +25,17 @@ EDISON-PATCHES = "\
 "
 
 SRC_URI = "${@bb.utils.contains('DISTRO_FEATURES', \
-	'acpi', 'git://github.com/htot/u-boot.git;branch=acpi-v2018.03;protocol=https', \
+	'acpi', 'git://github.com/htot/u-boot.git;branch=acpi-v2018.05;protocol=https', \
```

Comment:
Andy released his new 2018-05. I tested by building and installing the version with acpi, using the rocko64-acpi branch. I did not yet test the no-acpi version (rocko32 branch)